src/scancode/cli.py

```diff
@@ -458,11 +465,19 @@ def scancode(ctx,
             full_root=full_root,
             ignore=user_ignore)
 
+        for option, process in plugincode.post_scan.get_post_scan_plugins().items():
+            if kwargs[option.replace('-', '_')]:
+                options['--' + option] = True
+                if not quiet:
+                    echo_stderr('Running post-scan plugin: %(option)s...' % locals(), fg='green')
+                results = process(scanners, results, options, input)
```

Comment:
Are you calling process once for each option? I am not sure Get the calling logic here. Where are the post scan plugins called vs where is the main call the run the "scanners"?